### ISSUE TITLE
gnome 45 support

### ISSUE THREAD

**Issue by Cristiandis** [yilozt/rounded-window-corners#142]

**Comment by martijk** [comment]
See https://github.com/yilozt/rounded-window-corners/issues/130.

**Comment by DavidWoli** [comment]
@martijk Yeah I know, but just because it is merged into the `main` branch, doesn't mean that the gnome 45 support is there for end user.
@yilozt Still have to release a new version to gnome extensions store.

**Comment by deusnovus** [comment]
Still not on GNOME Extensions store...

**Comment by 42Willow** [comment]
For now just use the artifacts at https://github.com/yilozt/rounded-window-corners/actions

**Comment by deusnovus** [comment]
> For now just use the artifacts at https://github.com/yilozt/rounded-window-corners/actions

How do I do that? (I agree, I also don't feel like installing 10 packages just for one extension)

**Comment by 42Willow** [comment]
click on the latest build, download the zip, extract it into `~/.local/share/gnome-shell/extensions` and remove .shell-extension from the folder name so to match the UUID in the manifest

**Comment by deusnovus** [comment]
That works great, thank you so much. What I did yesterday was to download the .zip from the latest main branch, but didn't know I had to remove the `.shell-extension` from the name before moving it to `/extensions`.

**Comment by basilioss** [comment]
On Arch Linux you can install `gnome-shell-extension-rounded-window-corners-git` from [AUR](https://aur.archlinux.org/packages/gnome-shell-extension-rounded-window-corners-git)

**Comment by marcinjahn** [comment]
Is the Gnome 45 version already submitted for the EGO review?

**Comment by imotos** [comment]
Gnome 45 update? @yilozt

**Comment by mikoducky** [comment]
> On Arch Linux you can install `gnome-shell-extension-rounded-window-corners-git` from [AUR](https://aur.archlinux.org/packages/gnome-shell-extension-rounded-window-corners-git)

This worked great for me on Manjaro. Thanks!

**Comment by sihawido** [comment]
@yilozt Since the extension works fine on Gnome 45 are there any plans to put it on extensions.gnome.org?

**Comment by rivenirvana** [comment]
flexagoon's fork is now up on EGO, but is only for GNOME 46. https://extensions.gnome.org/extension/7048/rounded-window-corners-reborn/
